test/functional/find-options/basic-usage/find-options-test-utils.ts

```diff
@@ -82,4 +82,15 @@ export async function prepareData(manager: EntityManager) {
     post3.counters.likes = 1
     post3.counters.likedUsers = [user2]
     await manager.save(post3)
+
+    const post4 = new Post()
+    post1.id = 4
+    post1.title = "Post #4"
+    post1.text = "About post #4"
+    post3.author = user1
+    post1.tags = []
+    post1.counters = new Counters()
+    post1.counters.likes = 1
+    post1.counters.likedUsers = [user1]
```

Comment:
Should be `post4` instead of `post1` and `post3`.

Reply:
thank you!